### DISCUSSION TITLE
Meeting Minutes May 18, 2023

### DISCUSSION THREAD

**Discussion by jialincheoh** [opendigital/opendigital-org#29] [General]
**Marketing Effort**

Website publication needs to be fixed. @twinblade02 @jialincheoh

**CLA workshop**

Create issues @ManthanKeim @jialincheoh 

**Powerpoint Design**

Currently on pause @crashidian

**Access Issue**

Lionel is added to opendigital-org repo.

**Others**

Christopher visiting on June/ July

**Comment by jasminebui1225** [comment]
# Windracers's Chairman Visit (next week) 
- [ ] Manthan: Share the agenda + come in-person
- [ ] Jasmine: put this as news

# Represent strategic areas on website: 
- Conversational AI, IronHacks, Human teaming, Supply chain. We need to represent these areas better as strategic areas.
- [ ] Everyone: Look over Zotero & bib. files after the meeting. Update Zotero every week. (Jia is working on this).
- Issue: duplication of publications on the website (although not appearing on Zotero)

# IronHacks
- Follow up on CLA. 
- Press release -> social media.
